library/src/main/java/com/leinardi/android/speeddial/SpeedDialView.java

```diff
@@ -311,6 +319,7 @@ public void addActionItem(SpeedDialActionItem actionItem, int position, boolean
      * @param position the index of the {@link SpeedDialActionItem} to be removed
      * @return the {@link SpeedDialActionItem} that was removed from the list
      */
+    @SuppressWarnings("UnusedReturnValue")
```

Comment:
Please add the `"UnusedReturnValue"` to the class annotation and remove these single annotations.

- [ ] move suppression to class